packages/components/src/server-side-render/index.js

```diff
@@ -109,7 +109,7 @@ export class ServerSideRender extends Component {
 				<Placeholder
 					className={ className }
 				>
-					{ __( 'No results found.' ) }
+					{ __( 'Block rendered an empty area.' ) }
```

Comment:
Is `area` a good choice to describe this case?

Reply:
Some thoughts, what about "Block rendered as empty" or no text at all if it's empty ?
If on the contrary you want to emphasize that an empty render shouldn't happen then maybe "The block failed to render"

Reply:
I updated to use the string "Block rendered as empty", I also think not rendering anything is an option to consider, but as blocks may not expecting it, I thought to add a message is the safest bet.